backup.sh

```diff
@@ -15,23 +16,24 @@ copy_s3 () {
   echo "Uploading ${DEST_FILE} on S3..."
   if [[ -z "$S3_PREFIX" ]]; then
     # backup without prefix
-    aws $AWS_ARGS s3 cp $SRC_FILE s3://$S3_BUCKET/$DEST_FILE && \
-    aws $AWS_ARGS s3 cp s3://$S3_BUCKET/$DEST_FILE s3://$S3_BUCKET/$DEST_LATEST_FILE
+    aws $AWS_ARGS s3 cp $SRC_FILE s3://$S3_BUCKET/$FREQUENCY/$DEST_FILE && \
```

Comment:
@vijayabharathib did we test the restore after the backup.?

Reply:
tested the restore based on latest hourly backup . it worked. both for images and also for db. 

after backup, i inserted an update in wiki, after restore , my latest update was gone . this is what i expected. 
for images, i had 25 in the backup. after backup deleted 3. after restore i see 25 again. all good.

Reply:
@vijayabharathib The latest will work, Because that is there. 

Currently we store the the backup in different folder right?

Did we test the restore from there? 

For ex: restore from weekly/monthly backup.